### PR TITLE
Change text style does not invoke auto scroll

### PR DESCRIPTION
**Pull request by ArtifexEt**
Added new attribute that extends observer config to watch element attributes changes.

`<div class="list" angular2-auto-scroll lock-y-offset="10" observe-attributes >...</div>
`

### CODE REVIEW

File: src/angular2-auto-scroll.directive.ts [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
     @Input('angular2-auto-scroll') lockYOffset = 10;
 
+    @Input() set observeAttributes(observeAttributes: boolean) {
+        this._observeAttributes = observeAttributes || false;
+    }
+
     private nativeElement: HTMLElement;
     private isLocked = false;
     private mutationObserver: MutationObserver;
+    private _observeAttributes: boolean = false;
```
Comment (NagRock) [inline src/angular2-auto-scroll.directive.ts]: Please don't use under score prefix

File: src/angular2-auto-scroll.directive.ts [hunk 2]
```diff
@@ -25,7 +31,11 @@ export class Angular2AutoScroll implements AfterContentInit, OnDestroy {
                 this.nativeElement.scrollTop = this.nativeElement.scrollHeight;
             }
         });
-        this.mutationObserver.observe(this.nativeElement, {childList: true, subtree: true});
+        this.mutationObserver.observe(this.nativeElement, {
+            childList: true,
+            subtree: true,
+            attributes: this._observeAttributes
```
Comment (NagRock) [inline src/angular2-auto-scroll.directive.ts]: You should update readme with information about this new attribute.

**Review by NagRock — CHANGES_REQUESTED** [review]

File: src/angular2-auto-scroll.directive.ts (hunk 1 above)
Comment (ArtifexEt) [reply src/angular2-auto-scroll.directive.ts]: done

File: src/angular2-auto-scroll.directive.ts (hunk 2 above)
Comment (ArtifexEt) [reply src/angular2-auto-scroll.directive.ts]: done

**Review by NagRock — APPROVED** [review]

**Pull request merged**